packages/create-block/lib/init-block.js

```diff
@@ -65,15 +68,17 @@ async function initBlockJSON( {
 
 module.exports = async function ( outputTemplates, view ) {
 	await Promise.all(
-		Object.keys( outputTemplates ).map(
-			async ( outputFile ) =>
-				await writeOutputTemplate(
-					outputTemplates[ outputFile ],
-					join( view.folderName, outputFile ),
-					view
-				)
-		)
-	);
+		Object.keys( outputTemplates ).map( async ( outputFile ) => {
+			const pathName = ! view.plugin
+				? join( process.cwd(), view.slug, outputFile )
+				: join( view.folderName, outputFile );
```

Comment:
I haven't tested it but for consistency we might want to include the full path here as well:
```suggestion
				: join( process.cwd(), view.folderName, outputFile );
```

Reply:
Updated.

Reply:
I had to revert this as it seemed to be creating a `Users/path/to/where/I/called/it` directory in the `src` when scaffolding.

Reply:
Good 👍